src/main/java/core/basesyntax/service/operation/OperationHandler.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.Map;
+
+public interface OperationHandler {
+    Map<String, Integer> getOperation(FruitTransaction transaction);
```

Comment:
The method name `getOperation` might be misleading. Consider renaming it to something more descriptive, like `execute` or `performOperation`, to better reflect its purpose of performing an operation.